Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6-stretch
```

Comment:
Produkční verze a testovaná verze je 3.4, tak bych ji nechal raději i tady.

Reply:
Je to tam :) Máš pravdu, že to tak bude lepší.